Cargo.toml

```diff
@@ -31,6 +31,7 @@ __doc = []
 cfg-if = "0.1.10"
 rand = { version = "0.7", optional = true, default-features = false }
 serde = { version = "1", optional = true, default-features = false, features = ["derive"] }
+_serde_json = { package = "serde_json", version = "1", optional = true }
```

Comment:
Since optional dev-dependencies aren't possible, I'd prefer this be left out and just ` ```ignore ` the test. Hopefully some day it'll be allowed.